src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -27,6 +27,7 @@
 import com.cloudbees.jenkins.GitHubWebHook;
 import com.cloudbees.plugins.credentials.CredentialsNameProvider;
 import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
Please do not use this annotation. In general avoid any dependency on either Guava or Google Commons in plugins. Those libraries do not respect the same backwards compatibility contract of Jenkins core.